src/camvidlog2/vid.py

```
@@ -213,6 +214,69 @@ def generate_frames_cv2_rtsp(
         video_capture.release()
 
 
+def _generate_latest_frames_cv2_rtsp(
+    video_capture: cv2.VideoCapture,
+    target: np.ndarray,
+    lock: threading.Lock,
+    terminator=threading.Event,
```

Ruff (0.11.9)

Missing type annotation for function argument `terminator`

(ANN001)